examples/qencoder/qe_main.c

```diff
@@ -242,7 +242,7 @@ int main(int argc, FAR char *argv[])
     {
       printf("qe_main: open %s failed: %d\n", g_qeexample.devpath, errno);
       exitval = EXIT_FAILURE;
-      goto errout_with_dev;
+      goto errout;
```

Comment:
I think this line is actually the only change needed. The open fails we should just take the error path with no close. In the other cases we should take the path with the close.

Reply:
Done, thanks.